packages/ui/src/components/collectibles/native/collectible-card.native.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { CollectibleCard } from './collectible-card.native';
+
+const meta: Meta<typeof CollectibleCard> = {
+  title: 'Collectibles/CollectibleCard',
+  component: CollectibleCard,
+  tags: ['autodocs'],
+  argTypes: {},
+  parameters: {},
+  decorators: [Story => <Story />],
+};
+
+export default meta;
+
```

Comment:
I added some mocks of what we use to actually render inscriptions here.

- Video seems to be handled the same as `html` / `svg` in an iframe anyway. 
- Audio seems to just [show an icon](https://github.com/leather-io/extension/blob/dev/src/app/features/collectibles/components/_collectible-types/collectible-audio.tsx) now 

 I can inscribe audio + video on ordinals bot if we think that's useful. For video the network fee is high so it's ~ $392 for what I tested. Audio can be cheaper ~ $122.02.